Assets/Scripts/UI/DialogBox/Settings/DialogBoxSettings.cs

```diff
@@ -100,6 +107,10 @@ public void OnLangageToggle()
         langDropDown.SetActive(langToggle.isOn);
     }
 
+    public void OnDeveloperModeToggle()
+    {
+    }
```

Comment:
Honestly I only put it there for consistency with everything else. Removing it now.